### PR TITLE
fix: slow/fast disk stateful set removal, benchmark adjustments

### PR DESCRIPTION
**Pull request by Matovidlo**
**Changes:**
- Remove 2 disks from stateful set.
- Adjust k6 script to be able to add `port` into URL
- Add node and pod affinity for benchmark deployment.

-----------

### CODE REVIEW

File: provisioning/stream/kubernetes/templates/benchmark/job.yaml [hunk 1]
```diff
@@ -19,6 +20,23 @@ spec:
...
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/hostname
+                operator: NotIn
+                values:
+                - gke-dev-keboola-gcp-us-central-main-a-c6968d17-jm8r
+                - gke-dev-keboola-gcp-us-central-main-a-c6968d17-6zmw
```
Comment (michaljurecko) [inline provisioning/stream/kubernetes/templates/benchmark/job.yaml:32]: Please, add a comment.

Comment (michaljurecko) [reply provisioning/stream/kubernetes/templates/benchmark/job.yaml:32]: Also to the `podAntiAffinity`.

**Review by michaljurecko — APPROVED** [review]

**Comment by michaljurecko** [comment]
What will be done with this PR?
https://github.com/keboola/keboola-as-code/pull/1991

**Comment by Matovidlo** [comment]
@michaljurecko I think we should merge it since we do not have proposal how to do persistent benchmarking of services. We would like to build the `k6` in CI and push it to registry. We had discussion where should this script be. It can be less universal and built within each github repository or we would like to have it in some `tools` repository, where the k6 script has connection to other services as well.

File: provisioning/stream/kubernetes/templates/benchmark/job.yaml (hunk 1 above)
Comment (Matovidlo) [reply provisioning/stream/kubernetes/templates/benchmark/job.yaml:36]: Fixed and comments are added.

**Pull request merged**